### PR TITLE
libesedb 20240420 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: audit_exceptions/github_prerelease_allowlist.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "grt": "0.2.4",
   "joshuto": "all",
   "lha": "all",
+  "libesedb": "all",
```
Comment (SMillerDev) [inline audit_exceptions/github_prerelease_allowlist.json:9]: Why would we want to make an exception for this software?

Comment (chenrui333) [reply audit_exceptions/github_prerelease_allowlist.json:9]: so far all the releases are all marked as pre-release

image

Comment (chenrui333) [reply audit_exceptions/github_prerelease_allowlist.json:9]: I could just change to allow the current ones though, also gonna raise an ticket about it.

**Comment by chenrui333** [comment]
I will just allow 20240420 for now.

**Review by daeho-ro — APPROVED** [review]

**Review by nandahkrishna — APPROVED** [review]

**Pull request merged**